src/Engines/MobXEngine.js

```diff
@@ -0,0 +1,5 @@
+import { observer } from 'mobx-react/native';
+
+export default {
+  appHoc: observer,
```

Comment:
Right now I allow passing a HOC. I THINK that should be enough but I am not sure yet. I want to make this as uncomplex as possible while still being open enough to work for all (or at least most...) state implementations. I think this could work for Redux because you just pass it the `connect` HOC with a good mapStateToProps and mapDispatchToProps if needed.